src/common/lang.js

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft. All rights reserved.
 
 const lang = {
-  ACKNOWLEDGE: 'Acknowledge',
+  ACKNOWLEDGE: 'Ack',
```

Comment:
The name should match the string as much as possible so either change  `ACKNOWLEDGE` to `ACK` or add `ACK` as a key in the map.